.github/workflows/ci-aarch64.yml

```diff
@@ -174,6 +183,55 @@ jobs:
           CTEST_PARALLEL_LEVEL: 6
           DYLD_LIBRARY_PATH: ${{ github.workspace }}/ComputeLibrary/build
           ONEDNN_THREADING: ${{ matrix.config.threading }}
+
+      ## Regression test steps ##
```

Comment:
Please change the term to 'performance test' or 'performance regression test' to avoid confusion with functional regression tests.